doc/py_tutorials/py_feature2d/py_brief/py_brief.markdown

```diff
@@ -55,11 +57,11 @@ from matplotlib import pyplot as plt
 
 img = cv2.imread('simple.jpg',0)
 
-# Initiate STAR detector
-star = cv2.FeatureDetector_create("STAR")
+# Initiate FAST detector
```

Comment:
no, since STAR is in contrib, i thought of using fast instead, forgot to change comment back.